src/current/_includes/releases/cloud/2022-11-07.md

```diff
@@ -14,7 +14,6 @@
 
 - Added an icon next to a cluster's name on the [**Billing overview**](https://www.cockroachlabs.com/docs/cockroachcloud/billing-management) page to indicate when a cluster has been deleted.
 - The [**Database page**](https://www.cockroachlabs.com/docs/cockroachcloud/databases-page) in the CockroachDB {{ site.data.products.cloud }} Console now shows the last time table statistics were updated.
-- All new AWS clusters now use [`gp3` volumes](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/general-purpose.html). Previously created AWS clusters still use `io1` volumes. AWS `gp3` volumes expose three parameters: storage amount, IOPS, and throughput.
```

Comment:
Can you also remove note  with the mention of DSv4 VMs
https://www.cockroachlabs.com/docs/releases/cloud#2023-09-08-general-changes